newIDE/app/src/EventsSheet/index.js

```diff
@@ -1502,6 +1504,7 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
                         serializedEventsToExtract: null,
                       })
                     }
+                    onOpen={onBeginCreateEventsFunction}
```

Comment:
I would avoid `onOpen` (the parent is the one responsible for the child, so they should know when they open it) and add the call to onBeginCreateEventsFunction in `extractEventsToFunction`

Reply:
True, I will change that